modules/currency-l0/src/main/scala/org/tessellation/currency/l0/modules/Services.scala

```diff
@@ -89,6 +92,8 @@ object Services {
         maybeDataApplication
       )
 
+      sentStateChannelBinaryTrackingService <- SentStateChannelBinaryTrackingService.make[F](storages.identifier)
```

Comment:
It seems that it is created twice in the same flow, why? 🤔

Reply:
Fair point, leftover with variable shadowing